tests/installation/system_role.pm

```diff
@@ -37,11 +37,13 @@ sub change_system_role {
             send_key_until_needlematch "system-role-$system_role-selected", 'spc';     # enable role
         }
         else {
-            # scroll down in case role is not visible straight away
-            send_key_until_needlematch "system-role-$system_role", 'down', 5, 1;
-            assert_and_click "system-role-$system_role";
-            # after role is selected, we get top of the list shown, so might need to scroll again
-            send_key_until_needlematch "system-role-$system_role-selected", 'down', 5, 1;
+            if (!check_screen("system-role-$system_role")) {
```

Comment:
I guess we should use `system-role-$system_role-selected` and in case it's not - select it. Or did I misunderstand the motivation?

Reply:
Sure, I was confused by the filename instead of the tag name. Let me fix it.